cups/adminutil.c

```diff
@@ -698,8 +700,11 @@ cupsAdminSetServerSettings(
   if (server_port <= 0)
     server_port = IPP_PORT;
 
-  while (cupsFileGetConf(cupsd, line, sizeof(line), &value, &linenum))
+  while (_cupsFileGetConfAndComments(cupsd, line, sizeof(line), &value, &linenum, comment, sizeof(comment)))
```

Comment:
What about returning the comment via ```line``` parameter and remove the new additional parameters? Then the comment printout can happen via last cupsFilePrintf(). WDYT?